pypsa/network/index.py

```diff
@@ -757,3 +759,84 @@ def scenario_weightings(self) -> pd.DataFrame:
     def has_scenarios(self) -> bool:
         """Boolean indicating if the network has scenarios defined."""
         return len(self._scenarios_data) > 0
+
+    # -----------
+    # Risk Preferences (CVaR)
+    # -----------
+
+    def set_risk_preference(self, alpha: float, omega: float) -> None:
+        """Set risk aversion preferences for stochastic optimization using CVaR formulation.
+
+        Parameters
+        ----------
+        alpha : float
+            Confidence level in (0, 1). CVaR averages losses over the worst
+            (1 - alpha) probability mass (the "tail"). For "worst 10%",
+            set alpha = 0.9 so that 1 - alpha = 0.1. Typical choices
+            are alpha in {0.90, 0.95, 0.99}. Higher alpha focuses on
+            rarer, more extreme tails; lower alpha considers a broader tail.
+        omega : float
+            Risk preference parameter (risk aversion weight). Must be between 0 and 1.
+            - omega = 0: Risk-neutral optimization
+            - omega > 0: Risk-averse optimization (more focus on the tail risk)
+            - omega = 1: Maximum risk aversion (optimize for the tail risk only)
+            Higher values indicate more risk aversion.
+
+        Raises
+        ------
+        ValueError
+            If alpha is not between 0 and 1, or if omega is not between 0 and 1.
```

Comment:
```suggestion
            If alpha is not in (0,1), or if omega is not in [0,1].
```
To emphasize closed versus open interval.